x-pack/qa/runtime-fields/build.gradle

```diff
@@ -101,7 +99,6 @@ subprojects {
           'search/330_fetch_fields/error includes glob pattern',
           // we need a @timestamp field to be defined in index mapping
           'search/380_sort_segments_on_timestamp/*',
-          'field_caps/40_time_series/*',
```

Comment:
I wonder if this is a bit broad. How many are failing without this?

Reply:
The yaml failed because of the setup create index, index mapping has `time_series_dimension` and `time_series_metric`.
I remove this line, and run the test, it is ok.

Reply:
Could you remove the line from the PR? I think you've covered this in the translater.

Reply:
ok, done